demo/Dockerfile.alpine

```
@@ -63,24 +63,25 @@ RUN  apk update \
        python3 \
        ruby \
        ruby-rake \
-  && PIP_ROOT_USER_ACTION='ignore' pip install conan -v 'conan==2.0.9'
+  && PIP_ROOT_USER_ACTION='ignore' pip install conan
 
 # make an agent directory and cd into it
 WORKDIR /root/agent
 
 # bring in the repo contents, minus .dockerignore files
-COPY . .
+COPY . cppagent
 
 ARG WITH_TESTS=false
 ARG WITH_TESTS_ARG=argument
 
-# Build and optionally test
+# Build and optionally test. Unpack the dist to reduce overhead in the following release step.
 RUN  if [ -z "$WITH_TESTS" ] || [ "$WITH_TESTS" = "false" ]; then \
        WITH_TESTS_ARG="--test-folder="; \
      else \
        WITH_TESTS_ARG=""; \
      fi \
   && conan profile detect \
+  && cd cppagent \
   && conan create . \
```

I believe we could eliminate changing directory by replacing `.` with an absolute path.

```suggestion
  && cd cppagent \
  && conan create /root/agent/cppagent \
```

---

we could also use relative since the workdir is already set

---

Indeed, we could; like this:

```suggestion
  && conan create cppagent \
```